Update joins.mdx (#575)
risingwavelabs/risingwave-docs: processing/sql/joins.mdx

<Note>
- There is currently no way to disable the log store in-place.
- This optimization introduces a trade-off: improved pipeline stability at the cost of increased end-to-end latency due to the buffering step. Monitoring for potential data lag in the buffer is recommended.

> - High join amplification means there's a large amount of rows produced from the join. While barrier latency is mitigated with this approach, we still need to scale the cluster, to provide more resources for processing the large amount of rows.

Reviewer: ```suggestion
- High join amplification means a large number of rows are produced from the join. While this approach mitigates barrier latency, we still need to scale the cluster to provide more resources for processing the large amount of rows.
```
